testsuite/libra-fuzzer/README.md

```diff
@@ -56,3 +56,11 @@ are running a single test, set `NO_FORK=1` to disable forking.
 Once the deserializer has been fixed, check the artifact into the
 `artifacts/<target>/` directory. The artifact will then act as a
 regression test in `cargo test` runs.
+
+### Fuzzing Coverage
+
+To test coverage of our fuzzers you can run the following command with [tarpaulin](https://crates.io/crates/cargo-tarpaulin):
```

Comment:
`cargo-tarpaulin` only supports Linux at the moment. Might want to put a note here remind folks trying to run it on Mac.

Reply:
is it necessary here? 1) they might change this and make it available for mac later, 2) it's pretty obvious if you try to use tarpaulin that it's not supported for mac.